App.js

```diff
@@ -1,19 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import LogoSvg from "./LogoSvg";
-import Quotes from "./Quotes";
 import AnimateSvg from './AnimateSvg';
 import AnimateQuotes from './AnimateQuotes';
+import { useIsShake } from './useIsShake';
```

Comment:
Custom hook now utilized here in top level component and invoked once so that all child components will have access to single hook invocation.

Reply:
🦸 🦸 🦸